tests/unit/peapods/runtimes/asyncio/rest/test_models.py

```diff
@@ -253,7 +253,7 @@ def test_jina_document_to_pydantic_document_sparse():
             == pydantic_doc.embedding.sparse.values.buffer.decode()
         )
         assert (
-            jina_doc['embedding']['sparse']['values']['shape']
+            list(map(int, jina_doc['embedding']['sparse']['values']['shape']))
```

Comment:
why is this needed?

Reply:
@JoanFM after type changed, the `shape` is str, while the shape passed to pandantic is int

Reply:
why `shape` is in str?

Reply:
yes,  why is that?

Reply:
the shape in proto is a repeated int64

Reply:
@JoanFM @hanxiao 
> As per proto3 JSON spec, uint64/int64 fields should be printed as decimal strings. See:
https://developers.google.com/protocol-buffers/docs/proto3#json

The reason is that uint64/int64 is not part of JSON spec and many JSON libraries only support double precision. To prevent precision loss our proto3 JSON spec requires serializers to put int64/uint64 values in strings.

checkout [here](https://github.com/protocolbuffers/protobuf/issues/2954#issuecomment-292671661)

Reply:
another option is to use `int32` to represent the `shape`, how do you think @nan-wang ?

Reply:
uint32 it is